projects/longevity/data_gen/environment.yaml

```diff
@@ -1,4 +1,4 @@
-name: aframe-longevity
+name: aframe-datagen
```

Comment:
Is this now identical to the environment in `sandbox`?

Reply:
Yeah was kind of a trick to not have to rebuild since now that we've added `omicron` the environments are identical